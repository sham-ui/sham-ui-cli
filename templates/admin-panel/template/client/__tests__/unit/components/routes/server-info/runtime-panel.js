import RuntimePanel  from '../../../../../src/components/routes/server-info/runtime-panel.sfc';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( RuntimePanel, {} );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
