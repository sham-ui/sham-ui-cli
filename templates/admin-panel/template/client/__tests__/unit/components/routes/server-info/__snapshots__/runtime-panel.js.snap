// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Constructor": "RuntimePanel",
  "Options": Object {
    "ID": "component",
    "host": "",
    "runtime": Object {},
  },
  "html": "
<div class=\\"panel runtime\\">
  <h6>Runtime info</h6>
  <table>
    <tbody>
      <tr>
        <td>Host</td>
        <td></td>
      </tr>
      <tr>
        <td>Num CPU</td>
        <td>null</td>
      </tr>
      <tr>
        <td>Memory</td>
        <td>0 B</td>
      </tr>
      <tr>
        <td>Mem Sys</td>
        <td>0 B</td>
      </tr>
      <tr>
        <td>Heap Alloc</td>
        <td>0 B</td>
      </tr>
      <tr>
        <td>Heap Sys</td>
        <td>0 B</td>
      </tr>
      <tr>
        <td>Heap Idle</td>
        <td>0 B</td>
      </tr>
      <tr>
        <td>Heap Inuse</td>
        <td>0 B</td>
      </tr>
      <tr>
        <td>Next GC</td>
        <td>0 B</td>
      </tr>
      <tr>
        <td>Goroutines</td>
        <td>null</td>
      </tr>
      <tr>
        <td>UpTime</td>
        <td>NaN days NaN hours NaN minutes NaN seconds</td>
      </tr>
      <tr>
        <td>Time</td>
        <td>null</td>
      </tr>
    </tbody>
  </table>
</div>
",
}
`;
